### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by DanielDugan**
Update requirements.txt with requests==2.20

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Django==1.9
 mysql-python==1.2.5
 python-ldap==2.4.27
-requests==2.11.1
+requests==2.20
```
Comment (zqian) [inline requirements.txt]: Can we use the version style as the one in student-dashboard-django project, and mark this as 

requests>=2.20,<2.20.99

Comment (DanielDugan) [reply requirements.txt]: Absolutely! 
Do we want to change the nomenclature of all the dependencies in the requirements.txt file to assume this broad match for version control?

Comment (zqian) [reply requirements.txt]: Let's do the request change only this time, to address the security alert.

I think you can leave the change as "requests==2.20" for now, and open a separate issue for rewrite the all dependencies with new nomenclature format. Sorry about the confusion.

**Comment by DanielDugan** [comment]
I am closing this and opening a new one to update requirements.txt.

**Pull request closed without merging**